spec/controllers/forgot_passwords_controller_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'spec_helper'
+
+describe ForgotPasswordsController do
+  describe 'POST #create' do
+    before { ActionMailer::Base.deliveries.clear }
+
+    context "when users email is present in the database" do
+      let(:jane) { create(:user, email:'jane@example.com') }
+      it "sends email" do
+        post :create, email: jane.email
+        expect(ActionMailer::Base.deliveries).to_not be_empty
+      end
+
+      it "send email to the right user" do
+        post :create, email: jane.email
+        message = ActionMailer::Base.deliveries.last
+        expect(message.to).to eq([jane.email])
+      end
+
+      it "sends email with a link to the password reset page" do
+        post :create, email: jane.email
+        message = ActionMailer::Base.deliveries.last
+        expect(message.body.encoded).to match("http://localhost/reset_password/#{jane.token}")
```

Comment:
Yea i would probably go with `reset_password/#{jane.token}` the host can change in test later on.